Content.Shared/_RMC14/Clothing/HelmetAccesssoryHolderComponent.cs

```diff
@@ -9,6 +9,9 @@ public sealed partial class HelmetAccessoryHolderComponent : Component
 {
     [DataField, AutoNetworkedField]
     public SlotFlags Slot = SlotFlags.HEAD;
+
+    [DataField]
+    public bool IsHelmet = false;
```

Comment:
```suggestion
    public bool IsHelmet = false;
```
should probably rename this to IsHat for clarity? since the the other RSIs are called 'HatRsi', not 'HelmetRsi'

Reply:
i chose IsHelmet because it's for *helmet* garbs but yeah sure